onboarding-checklist.md

```diff
@@ -55,58 +56,40 @@ make sure you're assigned to this issue.
 #### Mandatory training ####
 
 - [ ] Complete Cybersecurity Training at the [CISA Course Management
-Site](https://usdhs.sharepoint.com/sites/CISA_OCLO_Intranet/CISACourses/),
-including Mandatory Cyber Security and Privacy Training, and
-acceptance of the Rules of Behavior, which are both required before we
-can give you access to any systems, including Trio.
+      website, including Mandatory Cyber Security and Privacy
+      Training, and acceptance of the Rules of Behavior, which are
+      both required before we can give you access to any systems,
+      including CISA GWE (Google Workspace Enterprise).
 
 ### Microsoft Teams chat ###
 
-Once you're set up on Microsoft Teams, we recommend you join the following:
+Once you're set up on Microsoft Teams, we recommend you join the
+following:
 
 - [ ] [CISA](https://teams.microsoft.com/l/team/19%3aa40fd338ea7a45de994a7c7ce9671fd4%40thread.skype/conversations?groupId=8c6c0b00-7244-46e1-bef9-76ce2a1906f5&tenantId=3ccde76c-946d-4a12-bb7a-fc9d0842354a)
 - [ ] [CSD All Staff](https://teams.microsoft.com/l/team/19%3a65f1c4b58edf4246995ae3fa732767ee%40thread.skype/conversations?groupId=6dfc3f23-3a49-44ce-ad37-6c7a9d8fb0aa&tenantId=3ccde76c-946d-4a12-bb7a-fc9d0842354a)
 - [ ] [Vulnerability Management](https://teams.microsoft.com/l/team/19%3a2161749c8eb94adeb31480e298f3b532%40thread.skype/conversations?groupId=61e939a6-2165-4085-a26e-decae97d1471&tenantId=3ccde76c-946d-4a12-bb7a-fc9d0842354a)
```

Comment:
Are we considering these internal links to be purged as well? It seemed like were just wholesale axing anything that isn't on GitHub.

Reply:
@cisagov/team-ois, what do you think?

Reply:
I'm fine getting rid of these since we don't really use them.

Reply:
@mcdonnnj, please see commit 30503ef.